src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Helper/QueryHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Community.PowerToys.Run.Plugin.ValueGenerator.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with queries
+    /// </summary>
+    internal static class QueryHelper
+    {
+        /// <summary>
+        /// A list that contain all of the value generators and its descriptions
+        /// </summary>
+        internal static readonly IReadOnlyDictionary<string, string> GeneratorList = new Dictionary<string, string>(18)
+        {
+            { "uuid", "Generate a random UUID" },
+            { "uuidv1", "Generate a version 1 UUID - Time based" },
+            { "uuidv3", "Generate a version 3 UUID - Namespace and name based, using MD5" },
```

Comment:
We should add input example to the title. Especially for the more complex features like uuidv3. Something like "uuidv3 <your domain> <your other input>".